phpseclib/Net/SCP.php

```diff
@@ -246,7 +246,10 @@ function get($remote_file, $local_file = false)
 
         $content = '';
         while ($size < $info['size']) {
-            $data = $this->_receive();
+            if (!$data = $this->_receive()) {
+                throw new \Exception('No data received from server');
```

Comment:
phpseclib 2.0 didn't throw any exceptions (outside of bootstrap.php, which is a little different). The phpseclib 2.0 way of doing this is to do `user_error`.

phpseclib 3.0 throws exceptions but it doesn't throw generic exceptions - the closest thing to a generic exception that it throws is prob [RuntimeException](https://www.php.net/manual/en/class.runtimeexception.php).

Also, from a CS perspective, I'd prefer it if you did the `if` statement more like this instead:

```php
$data = $this->_receive();
if (!$data) {
    // do stuff
}
```